src/com/nutrons/steamworks/Shooter.java

```diff
@@ -14,7 +17,7 @@ public Shooter(LoopSpeedController shooterController) {
 
   @Override
   public void registerSubscriptions() {
-    shooterController.setPID(0.05, 0.0, 0.33, 0.035);
-    shooterController.setSetpoint(2950.0);
+    Flowable<ControllerEvent> source = toFlow(() -> new LoopPropertiesEvent(2950.0, 0.05, 0.0, 0.33, 0.035));
+        source.mergeWith(toFlow(() -> new LoopModeEvent(ControlMode.LOOP_SPEED))).subscribe(shooterController);
```

Comment:
Also @totallyashes is adding imperative methods which might be the better choice here.